x-pack/test/apm_api_integration/tests/service_overview/dependencies/index.ts

```diff
@@ -323,14 +323,41 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
       it('returns at least one item', () => {
         expect(response.body.length).to.be.greaterThan(0);
+
+        expectSnapshot(
+          omit(response.body[0], [
+            'errorRate.timeseries',
+            'throughput.timeseries',
+            'latency.timeseries',
+          ])
+        ).toMatchInline(`
+          Object {
+            "errorRate": Object {
+              "value": 0,
+            },
+            "impact": 1.97910470896139,
+            "latency": Object {
+              "value": 1043.99015586546,
+            },
+            "name": "redis",
+            "spanSubtype": "redis",
+            "spanType": "db",
+            "throughput": Object {
+              "value": 40.6333333333333,
+            },
+            "type": "external",
+          }
+        `);
       });
 
       it('returns the right names', () => {
         const names = response.body.map((item) => item.name);
         expectSnapshot(names.sort()).toMatchInline(`
           Array [
-            "opbeans-go",
+            "elasticsearch",
+            "opbeans-java",
             "postgresql",
+            "redis",
```

Comment:
Why do these change? I thought the only change was to the impact calculation?

Reply:
I changed it to opbeans-python. opbeans-java only has two dependencies in our data set so it will always report one of 100 and one of 0.